src/clj/rems/application/master_workflow.clj

```diff
@@ -124,6 +127,7 @@
   [application event]
   (-> application
       (permissions/give-role-to-users :applicant [(:event/actor event)])
+      (permissions/give-role-to-users :expirer [expirer-bot/bot-userid])
```

Comment:
I guess it doesn't make sense to make this dependent on the config, as that would add dependency to this pure model to the config, as well as complicate the tests. It's sort of slightly confusing that even if expire isn't confirued, or `expirer-bot` used doesn't exist, it is added.

Reply:
good point. `rems.application.model/enrich-super-users` implements this already for `:reporter` role, so i moved the functionality there and added documentation about granting `:expirer` role manually to the Expirer bot.